app/src/main/java/kr/co/softcampus/sopt_assignment1/ui/activity/FindUserActivity.kt

```diff
@@ -20,11 +21,26 @@ class FindUserActivity : AppCompatActivity() {
         super.onCreate(savedInstanceState)
         binding = ActivityFindUserBinding.inflate(layoutInflater)
 
+        initcheckbox()
+        checkboxClickEvent()
         SearchButton()
 
         setContentView(binding.root)
     }
 
+    private fun initcheckbox() {
+        if (SharedPreferences.getAutoLogin(this)) {
+            binding.checkbox.isChecked = false
+        } else binding.checkbox.isChecked = true
+    }
```

Comment:
```suggestion
    private fun initcheckbox() {
       binding.checkbox.isChecked = !SharedPreferences.getAutoLogin(this)
       }
```
이렇게도 쓸 수 있을 것 같습니당~!!

Reply:
오 그렇네요! 훨씬 간단해질 수 있네요ㅎㅎ